datasets/batch_view/heavy_users/intro.md

```diff
@@ -0,0 +1,46 @@
+The `heavy_users` table provides information about whether a given client_id is
+considered a "heavy user" on each day (using submission date).
+
+#### Contents
+
+The `heavy_users` table contains one row per client-day, where day is
+submission_date. A client has a row for a specific submission_date if
+they were active at all in the 28 day window ending on that submission_date.
+
+A user is a "heavy user" as of day N if, for the 28 day period ending
+on day N, the sum of their active_ticks is in the 90th percentile (or
+above) of all clients during that period. For more analysis on this,
+and a discussion of new profiles, see
+[this link](https://metrics.mozilla.com/protected/sguha/heavy/heavycutoffs5.html).
+
+#### Background and Caveats
+
+1. Data starts at 20170801. There is technically data in the table before this, but heavy_user is NULL for those dates because it needed to bootstrap the first 28 day window.
+2. Because it is top 10% for each 28 day period, and single submission_date has more than 10% of clients be considered heavy_users. This is because heavy_users, on average, use Firefox on more days than non-heavy users.
```

Comment:
I can't seem to parse this sentence.

Reply:
@fbertsch this is your copy - can you offer a clarification? Or maybe an example?

Reply:
I definitely mistyped here originally.

`Because it is top the 10% of clients for each 28 day period, any single submission_date will have more than 10% of clients be considered heavy_users. This is because heavy_users, on average, use Firefox on more days than non-heavy users.`

If this isn't clear, consider the following:
10 users. The first 9 use Firefox on MWF only. The last one uses it every day, and is also the `heavy_user`. For 4/7 days a week, 100% of users are heavy_users!

Reply:
An alternative: `For clients active on a given submission_date, more than 10% will be heavy_users`

But this doesn't provide an explanation for "why".

Reply:
How's this:

> Because it is top the 10% of clients for each *28 day period*, more than 10% of clients *active* on a given submission_date will be considered heavy_users, so if you join with another data source (`main_summary`, for example), you may see a larger proportion of heavy users than expected.

Reply:
Looks good, only recommendation would be to split it into two sentences: `Because it is top the 10% of clients for each 28 day period, more than 10% of clients active on a given submission_date will be considered heavy_users. If you join with another data source (main_summary, for example), you may see a larger proportion of heavy users than expected.`